code/game/objects/items/devices/taperecorder.dm

```diff
@@ -489,12 +504,72 @@
 
 //Random colour tapes
 /obj/item/tape/random
-	icon_state = "random_tape"
+	icon_state = "cassette_flip"
 
 /obj/item/tape/random/Initialize(mapload)
 	icon_state = "cassette_[pick(cassette_colours)]"
 	. = ..()
 
+/obj/item/tape/random/loadout
+	desc = "A small plastic tape. Jams often."
+	max_capacity = 10 MINUTES
+
 /obj/item/tape/regulation
 	name = "regulation tape"
 	icon_state = "cassette_regulation"
+
+/*
+// HOW TO MAKE A CUSTOM AUDIO LOG TAPE
+// for spawning on maps or putting in the loadout tape recorder
+// done by example
+
+
+// typepath here, make sure to make it a subtype of audio_log
+/obj/item/tape/audio_log/example
+
+// do name desc and icon state here
+
+// these are the lines that will be said on the tape. Put them in the provided format for speech, and you can do whatever you like with other sounds
+// remember that you can make the recorder say ANYTHING
+	storedinfo = list(
+		"\[00:03] Dana Summy says, \"Okay\"" ,
+		"\[00:05] Alaina Suni says, \"Sure\"",
+		"\[00:06] Dana Summy says, \"Yes\"",
+		"\[00:08] *gunshots*",
+	)
+
+// these are the timestamps of the above messages, put in the same order as you put the timestamps ideally
+// if you need two messages to play immediately after each other make their timestamps the same
+// these timestamps are in TICKS (1/10ths of a second)
+
+	timestamp = list(
+		30,
+		50,
+		60,
+		80
+	)
+
+// how much of the tape has been used up
+// make this the biggest number on the timestamp list or 10 MINUTES if you don't want anything more to be recorded on the side.
+	used_capacity = 90
+
+
+// what the above tape produced when played in-game
+	Playback started.
+	[00:03] Dana Summy says, "Okay"
+	[00:05] Alaina Suni says, "Sure"
+	[00:06] Dana Summy says, "Yes"
+	[00:08] *gunshots*
+	End of recording.
+	Playback stopped.
+
+// have fun!
+
+*/
+
+/obj/item/tape/audio_log
+	name = "partially used tape" // RENAME!
+	desc = "A standard tape, made by the million in factories on Earth. This one has been partially used." // RENAME!
```

Comment:
```suggestion
	desc = "A standard tape, made by the million in factories on Terra. This one has been partially used." // RENAME!
```

calling earth terra is soulful and reinforces the idea that it's some sort of legendary place marines and colonists look up towards from their shithole vessels and colonies (even if they're wrong)

Reply:
it's not called that anywhere else + we are not TGMC (TERRAgov marine corps)